### DISCUSSION TITLE
Questions about CI/CD

### DISCUSSION THREAD

**Discussion by sali1982** [cypress-io/cypress#9382] [CI setup]
Hi,

I have a few questions.
1. how big are the video sizes? are they big?
2. Are videos saved in the running image? where we can find then and watch them if failed (Not using dashboards)
3. I assume time travel is not supported in the ci/cd? if I am wrong, how?

**Comment by geekiam23** [comment]
When you say time travel what do you mean? I think I came here looking for the same answer as you.

**Comment by sali1982** [comment]
Time travel is when you click on the lines in the browser to see the tests running.

Does anyone know how we can watch the videos on CI if we choose not to save on dashboard? where to find the videos if they fail?

**Reply by bahunov** [reply]
They'll be created under cypress/videos folder..same like cypress/screenshot
